backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/bayern/steps/WriteCsv.kt

```diff
@@ -0,0 +1,55 @@
+package app.ehrenamtskarte.backend.stores.importer.bayern.steps
+
+import app.ehrenamtskarte.backend.stores.importer.ImportConfig
+import app.ehrenamtskarte.backend.stores.importer.PipelineStep
+import app.ehrenamtskarte.backend.stores.importer.bayern.types.FilteredStore
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import org.apache.commons.csv.CSVFormat
+import org.slf4j.LoggerFactory
+import java.io.BufferedWriter
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class WriteCsv(config: ImportConfig, private val filteredStores: MutableList<FilteredStore>) :
```

Comment:
🔧 The name of this class is a little bit unspecific.

Reply:
renamed to `FilteredStoresCsvOutput`